src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -69,7 +62,7 @@ const ALIGN_CENTER = "center";
 
 const TOOLTIP_ARROW_OFFSET = 23;
 const TOOLTIP_FULLSCREEN_THRESHOLD = 480;
-const TOOLTIP_MAX_WIDTH = 366;
+const TOOLTIP_MAX_WIDTH = 320;
```

Comment:
Who provided this new value?

Reply:
it is current max tooltip content width (see [old css](https://github.com/gooddata/gooddata-react-components/pull/1147/files#diff-84b5bf43cbbf7caee4c0bdabb659f0efL119)). i just adjust the border width and padding as [mockup](https://gooddata.invisionapp.com/share/5MSHNVYCSUQ#/screens/368579121)